src/services/ethconversion/getEthConversionService.js

```diff
@@ -0,0 +1,84 @@
+const rp = require('request-promise');
+const logger = require('winston');
+
+const fiat = ['USD', 'EUR', 'GBP', 'CHF', 'MXN', 'THB'];
+const MINUTE = 1000 * 60;
+
+
+const _buildResponse = (timestamp, rates) => ({
+  timestamp: timestamp,
+  rates: rates
+})
+
+/**
+ Fetching eth conversion based on daily average from cryptocompare
+ Saves the conversion rates in the backend if we don't have it stored yet.timestamp
+
+ @params:
+    app: include the app object, won't work without for some reason
+    requestedData (Date): optional requested date, defaults to new Date()
+ **/
+
+export const getEthConversion = (app, requestedDate) => {
+  const date = requestedDate ? new Date(requestedDate) : new Date();
+  const startDate = date.setUTCHours(0,0,0,0);      // set at start of the day
+  const timestamp = Math.round(startDate) / 1000;   // create timestamp in seconds, as accepted by cryptocompare
+
+  logger.info(`request eth conversion for timestamp ${timestamp}`);
+
+  return new Promise((resolve, reject) => {
```

Comment:
you can just return the app.service call, as that is already a promise. no need to wrap everything in another promise..

any `resolve(...)` can just return the data & and `reject(...)` can throw

Reply:
ah yes :-)

Reply:
turns out there is a difference, so I'm keeping the promise